<?php

namespace ContainerIh461Mf;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_HsfYii4Service extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.hsfYii4' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.hsfYii4'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'callback' => ['privates', '.errored..service_locator.hsfYii4.Closure', NULL, 'Cannot autowire service ".service_locator.hsfYii4": it references class "Closure" but no such service exists. You should maybe alias this class to one of these existing services: "container.getenv", "data_collector.request.session_collector".'],
        ], [
            'callback' => 'Closure',
        ]);
    }
}
